bindings/rust/wasmedge-sys/src/instance/function.rs

```diff
@@ -58,8 +58,8 @@ extern "C" fn wraper_fn(
 #[derive(Debug)]
 pub struct Function {
     pub(crate) ctx: *mut wasmedge::WasmEdge_FunctionInstanceContext,
-    pub(crate) registed: bool,
-    ty: Option<Type>,
+    pub(crate) registered: bool,
+    pub(crate) ty: Option<FuncType>,
```

Comment:
And I would like to discuss with the naming of `Type`s. Do we have to add the prefix in each `Type` such as `FuncType`. Rust has namespace, would it be better to use `Func::Type` in this case?

Reply:
For now, wasmedge-sys involves several types: FuncType in function, MemType in memory, and ValType in types. We surely can distinguish these three by their namespace. However, when these types are used in the same code file and multiple times, that could become a little bit verbose, like function::Type. Of course, aliasing can help. But, if the case happens in several code files, users have to do it in each code file. In addition, (personal idea), the semantics implied by function::Type is not equal to the one by FuncType.